commit 7848789cbf006127bde475b15b79c5db1be6b2db
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:53:41 2010 -0700

    Staging: brcm80211: typedefs.h move types.h to the top of the file
    
    Include other files from the top, it's easier to unwind the logic that
    way.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -17,6 +17,8 @@
 #ifndef _TYPEDEFS_H_
 #define _TYPEDEFS_H_
 
+#include <linux/types.h>
+
 /*
  * Infer the compile environment based on preprocessor symbols and pragmas.
  * Override type definitions as needed, and include configuration-dependent
@@ -31,7 +33,6 @@ typedef unsigned long long int uintptr;
 #define TYPEDEF_UINT
 #define TYPEDEF_USHORT
 #define TYPEDEF_ULONG
-#include <linux/types.h>
 
 /*
  * Default Typedefs
